wallet/how-to/send-transactions/send-batch-transactions.md

```diff
@@ -14,7 +14,7 @@ You can send and manage batch transactions in MetaMask, using the methods specif
 ## About atomic batch transactions
 
 An atomic batch transaction is a group of transactions that are executed together as a single unit.
-When a dapp requests to submit a batch of transactions atomically, MetaMask may prompt users to upgrade their externally owned account (EOA) to a [MetaMask delegator account](https://docs.gator.metamask.io/concepts/delegator-accounts).
+When a dapp requests to submit a batch of transactions atomically, MetaMask may prompt users to upgrade their externally owned account (EOA) to a [MetaMask delegator account](https://docs.gator.metamask.io/concepts/delegator-accounts) (or "smart account").
 If the user accepts, MetaMask proceeds to upgrade the account and process the request as a single atomic transaction as specified by [EIP-7702](https://eips.ethereum.org/EIPS/eip-7702).
 
 :::note Delegator accounts
```

Comment:
Smart accounts?

Reply:
The note explains that delegator accounts are a type of smart account, and we do reference "delegator accounts" throughout this page, so I think it might be better to keep this title.